tools/cldr-apps/js/src/esm/cldrVettingParticipation.js

```diff
@@ -448,16 +476,37 @@ function getDefaultRow(id, user, columnIndex) {
 function addColumnComments(ws) {
   let unicode = "A".charCodeAt(0);
   for (let col of COLUMNS) {
-    const letter = String.fromCharCode(unicode++); // A, B, C, ... assume max 26 columns!
+    const letter = String.fromCharCode(unicode++); // A, B, C, ...
     const cell = letter + "1"; // A1, B1, C1, ...
```

Comment:
use the utility for this

```js
const cell = XLSX.utils.encode_cell({r: 1, c: column_number});
```

instead of `unicode` 

see https://docs.sheetjs.com/docs/csf/general#cell-addresses-1

Reply:
it's always fun to reinvent the wheel!

I'll change it to encode_cell in the next PR

Reply:
Actually "r: 0" not "r: 1"

The documentation is misleading

Reply:
sorry, yes r:0 my mistake